src/peripheral/README.md

```diff
@@ -55,4 +55,15 @@ ROS wrappers around wrp_sdk peripheral drivers.
 
 | Parameter     | Type   | Description                                                  |
 | ------------- | ------ | ------------------------------------------------------------ |
-| `device_path` | string | Path to power regulator can port.<br />Default: "can0"<br /> |
\ No newline at end of file
+| `device_path` | string | Path to power regulator can port.<br />Default: "can0"<br /> |
+
+### lift_controller_node
+
+| Service                   | Type               | Description                              |
+| ------------------------- | ------------------ | ---------------------------------------- |
+| `/lift_controller/service`| wrp_ros::LiftQuery | Query for state of the provided lift id. |
```

Comment:
Please add the action into the readme and maybe rename the service and action topics to be more descriptive

maybe 

/lift_controller/state_query
/lift_controller/goal